tests/unit/models/project-test.js

```diff
@@ -1,12 +1,69 @@
 import { moduleForModel, test } from 'ember-qunit';
+import Ember from 'ember';
 
 moduleForModel('project', 'Unit | Model | project', {
   // Specify the other units that are required for this test.
-  needs: ['model:project-category', 'model:organization', 'model:task', 'model:project-skill']
+  needs: ['model:project-category', 'model:organization', 
+          'model:organization-membership', 'model:user', 
+          'model:task', 'model:project-skill']
 });
 
 test('it exists', function(assert) {
   let model = this.subject();
   // let store = this.store();
   assert.ok(!!model);
 });
+
+test('it should has attributes', function(assert) {
+  let model = this.subject();
+  let attributes = Object.keys(model.toJSON());
+  
+  assert.ok(attributes.includes('base64IconData'), 'Has base64IconData attribute');
+  assert.ok(attributes.includes('closedTasksCount'), 'Has closedtasksCount attribute');
+  assert.ok(attributes.includes('description'), 'Has description attribute');
+  assert.ok(attributes.includes('iconLargeUrl'), 'Has iconLargeUrl attribute');
+  assert.ok(attributes.includes('iconThumbUrl'), 'Has iconThumbUrl attribute');
+  assert.ok(attributes.includes('longDescriptionBody'), 'Has longDescriptionBody attribute');
+  assert.ok(attributes.includes('longDescriptionMarkdown'), 'Has longDescriptionMarkdown attribute');
+  assert.ok(attributes.includes('openTasksCount'), 'Has openTasksCount attribute');
+  assert.ok(attributes.includes('slug'), 'Has slug attribute');
+  assert.ok(attributes.includes('title'), 'Has title attribute');
+});
+
+test('it should has relationships', function(assert) {
+  const Project = this.store().modelFor('project');
+  const relationships = Ember.get(Project, 'relationshipsByName');
```

Comment:
I've seen some debate on `const` vs `let`. Can you explain your reasoning for it?